screens/Home.js

```diff
@@ -1,53 +1,73 @@
 import React, {Fragment} from 'react';
-import Axios from 'react-native-axios';
+import API from '../API';
 import {
   SafeAreaView,
   Text,
-  StatusBar,
   FlatList,
+  TouchableOpacity,
+  Image,
+  StyleSheet,
+  AsyncStorage
 } from 'react-native';
 
-import CardProduct from '../components/CardProduct';
+import CardProduct from '../components/CardProductHome';
 
 class Home extends React.Component{
-  static navigationOptions = {
-      title: 'Loja virtual',
+  static navigationOptions = ({navigation}) => {
+      return {
+        title: 'Loja Virtual'
+      };
   }
 
-  constructor(){
-    super();
+  constructor(props){
+    super(props);
+
     this.state = {
       products: []
     }
   }
 
-  componentDidMount(){
-    Axios.get('http://9.232.24.93:8080/api/product')
-      .then(response => this.setState({products: response.data}))
-      .catch(err => this.setState({products: []}));
+  async componentDidMount(){
+    const products = await API.getProducts();
+    let aux = [];                               //
+    for(let i = 1; i < products.length; i++){  //apagar depois
```

Comment:
Para que serve esse `for`? Se for necessário, pq não usar um `Array.prototype.map`?